app/views/people/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Date Created</th>
+      <th>Date Updated</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><%= @person.name %></td>
+      <td><%= @person.created_at %></td>
```

Comment:
You can use strftime to format these dates: http://apidock.com/ruby/DateTime/strftime